bootstrap/eks/controllers/eksconfig_controller_test.go

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
WDYT about using the pattern in CAPI where the reconciliation tests are in a file called `eksconfig_controller_reconciler_tests.go` and leaving this filename for the unit tests?